### PR TITLE
PR2: Make epsilon for ZCA parameterizable

### PR DESCRIPTION
**Pull request by StefanoD**
Fixes [#6612](https://github.com/fchollet/keras/issues/6612)
This PR is an alternative to [PR #6614](https://github.com/fchollet/keras/pull/6614)

### CODE REVIEW

**Comment by taehoonlee** [comment]
Good!

File: keras/preprocessing/image.py [hunk 1]
```diff
@@ -346,6 +346,7 @@ class ImageDataGenerator(object):
         featurewise_std_normalization: divide inputs by std of the dataset.
         samplewise_std_normalization: divide each input by its std.
         zca_whitening: apply ZCA whitening.
+        zca_epsilon: epsilon for ZCA whitening. Default is 1e-6.
```
Comment (fchollet) [inline keras/preprocessing/image.py]: Why not use `K.espilon()` like everywhere else?

Comment (StefanoD) [inline keras/preprocessing/image.py]: As stated in the issue: A suitable epsilon depends on the scaling.

As [this](http://ufldl.stanford.edu/wiki/index.php/Data_Preprocessing) website states:
> Tip: If your data has been scaled reasonably (e.g., to [0,1]), start with epsilon = 0.01 or epsilon = 0.1. 

This epsilon is special to zca_whitening and might not be suitable for other cases where you need epsilon.

**Comment by fchollet** [comment]
LGTM

**Pull request merged**